### DISCUSSION TITLE
Center stack with one window

### DISCUSSION THREAD

**Discussion by bjoroen** [i3/i3#5825] [Q&A] [answered]
Hello everyone,

I am pretty new to i3 and window manager in general. I'm very happy, but there is one thing I just cant figure out how to do.

I have a ultra wide monitor, so I want a layout to start in the middle, 50% of the screen center.

So when I open the first window it starts like this:

| ------| Main |------ |

Then when I open an other window it turns to this: 

|------| Main | Stack |

Then if I open a third window: 

|  Stack  | Main | Stack |

Then other windows will join a stack, either one of them or alternate between them.

Ive google like crazy and cant find exactly what I`m looking for. If anyone knows how I can make this happen I would really appreciate any clues, is it possible to write something myself to make it happen if it isn't out there already?

**Comment by orestisfl** [comment, marked as answer]
The philosophy of the i3 window manager is to always utilize the full space of your monitor, this is not something you can do out of the box.

The part that automates opening two stacks and distributing windows between them you can implement this using the [IPC interface](https://i3wm.org/docs/ipc.html) with a script.

**Reply by bjoroen** [reply]
Thank you so much for the feedback. 
I did start to have my suspicions that this was going against the philosophy of most window managers. 
Thanks for the link to the IPC interface, ill implement that script.
